src/main/java/Task.java

```diff
@@ -0,0 +1,36 @@
+public class Task {
+    /* Task is a class, what does it do??? */
+    private String name; 
+    private boolean isDone = false;
+    private char taskType;
+
+    public Task(String name, String task) {
+        this.name = name;
+        this.taskType = task.charAt(0);
+    }
+
+    public void doneTask() {
```

Comment:
Perhaps you can change the method name to markTask which I feel would be a little clearer? 🤔